core/src/com/unciv/models/metadata/GameSettings.kt

```diff
@@ -140,6 +141,10 @@ class GameSettings {
     /** used to migrate from older versions of the settings */
     var version: Int? = null
 
+    /** NumberFormat cache. Key: Language, Value: NumberFormat **/
+    @Transient
+    val languageToNumberFormat = mutableMapOf<String, NumberFormat>()
```

Comment:
I'm not conviced this should sit in the GameSettings. This sounds like something for translation companion object, this and the getLocaleFromLanguage function

Reply:
I added this here because locale get or set functions was here. So, you are suggesting that we move `fun getLocaleFromLanguage(language: String): Locale`, `fun updateLocaleFromLanguage()`, fun `getNumberFormatFromLanguage(language: String): NumberFormat` and `fun getCurrentNumberFormat(): NumberFormat` functions to be moved to translations file?

Also not sure it I implement `Number.tr()` inside Translations class if it would be accessible to rest of the project. Fact: new to Kotlin.

Reply:
All of these apart from the "update locale" which is a settings thing (not sure why we need that locale but sure, probably for string sorting or something)

Reply:
Sorry not the "get current" either, that depends on current language which is setting

Reply:
`getLocaleFromLanguage()` also depends on `enum LocalCode` inside settings file.

Reply:
I guess only `fun getNumberFormatFromLanguage(language: String): NumberFormat` and `languageToLocale` map should be moved.

Reply:
resolved with c567e94